corehq/apps/app_manager/templates/app_manager/profile.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <profile version="{{ app.version }}"
-         update="{{ profile_url }}"
+         update="{{ profile_url }}&amp;profile={{ build_profile_id }}"
```

Comment:
yes to both. i posted the line where it happens in a reply to danny that i think has been covered up by an outdated diff, but the profile_url always ends in `?latest=true` and the other one is generated by django url reverse so wont have query parameters